nocodb/infra/requests_client.py

```diff
@@ -12,13 +13,33 @@
 
 
 class NocoDBRequestsClient(NocoDBClient):
-    def __init__(self, auth_token: AuthToken, base_uri: str):
+    def __init__(
+        self,
+        base_uri: str,
+        auth_token: AuthToken = None,
+        email: str = None,
+        password: str = None,
+    ):
         self.__session = requests.Session()
+        self.__api_info = NocoDBAPI(base_uri)
+
+        if not auth_token and not (email and password):
+            raise ValueError("Either сredentials or token must be provided")
+
+        if not auth_token and (email and password):
+            auth_token = JWTAuthToken(self.get_auth_token(email, password))
+
         self.__session.headers.update(
-            auth_token.get_header(),
-        )
+                auth_token.get_header(),
```

Comment:
This change has nothing to do with the commit.

```suggestion
            auth_token.get_header(),
```